diff -p -up gluezilla-2.6/configure.ac~ gluezilla-2.6/configure.ac
--- gluezilla-2.6/configure.ac~	2010-01-02 14:22:10.000000000 +0100
+++ gluezilla-2.6/configure.ac	2010-01-02 14:23:25.000000000 +0100
@@ -88,7 +88,7 @@ if test "x${local}" == xno ; then
 	if test "x${oldxul}" != xyes ; then
 
 		AC_MSG_CHECKING(Mozilla XPCOM > 1.8)
-		mozilla_xpcom_pcs="libxul-embedding-unstable"
+		mozilla_xpcom_pcs="libxul-embedding"
 		for pc in $mozilla_xpcom_pcs; do
 			if $PKG_CONFIG --exists $pc; then
 				AC_MSG_RESULT($pc)
